app/controllers/users/articles_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Users
+  class ArticlesController < ApplicationController
+
+    expose :articles, -> { current_user.articles.where(filter_params).order(created_at: :desc).page params[:page] }
+    expose :article, parent: :current_user
+
+    before_action :authenticate_user!
+    before_action :authorize_resource!, only: %i[edit update destroy]
+
+    def index
+    end
+
+    def show
+    end
+
+    def create
+      article.save
+      respond_with article, location: -> { user_article_path(article.id) }
+    end
+
+    def new
+    end
+
+    def edit
+    end
+
+    def update
+      article.update(article_params)
+      respond_with article, location: user_article_path(article.id)
+    end
+
+    def destroy
+      article.destroy
+      respond_with article, location: user_articles_path
+    end
+
+  private
+
+    def authorize_resource!
+      authorize article
+    end
+
+    def article_params
+      params.require(:article).permit(:title, :content, :status)
+    end
+
+    def filter_params
```

Comment:
in this case, we can apply query object pattern
I think we can discuss it on personal meeting